src/Workspaces/Core/Portable/Remote/RemoteServiceCallbackId.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Runtime.Serialization;
+
+namespace Microsoft.CodeAnalysis.Remote
+{
+    [DataContract]
+    internal readonly struct RemoteServiceCallbackId : IEquatable<RemoteServiceCallbackId>
```

Comment:
💭 I thought we already had this in another PR

Reply:
➡️ It's being moved here from another file

Reply:
This just moved to a separate file.